src/main.js

```diff
@@ -211,30 +191,92 @@ function filterIdeas(){
         } else {
           imgCardSrc = `src="./src/icons/star.svg"`
         }
-        var ideaCard = `
-          <article data-id="${list[i].id}" class="idea-card">
-            <section class="idea-card-header">
-              <img class="star-icon" ${imgCardSrc} alt="favorite Card">
-              <img class="delete-icon" src="./src/icons/delete.svg" alt="Delete Card">
-            </section>
-            <section class="idea-card-body">
-              <h3 class="idea-title">
-                ${list[i].title}
-              </h3>
-              <p class="idea-body">
-                ${list[i].body}
-              </p>
-            </section>
-            <section class="idea-card-footer">
-              <img class="add-icon" src="./src/icons/comment.svg" alt="Comment Icon">
-              <p class="comment">Comment</p>
-            </section>
-          </article>
-        `;
-        ideaCardsGrid.insertAdjacentHTML("afterbegin", ideaCard)
+      printIdeaHTML(list[i], imgCardSrc);
       }
     }
   } else {
     displayIdeaCards();
   }
 }
+
+function toggleCommentForm(){
+  var commentForm = event.target.parentNode.nextElementSibling;
+  commentForm.classList.toggle("hidden");
+  var button = event.target.parentNode.nextElementSibling.lastElementChild
+  button.disabled = true;
+}
+
+function displayComment(){
+  var commentSection = event.target.parentNode.nextElementSibling
+  var commentInput = event.target.previousElementSibling
+  var currentComment = new Comment(commentInput.value)
+  console.log(currentComment)
+  event.preventDefault();
+  var comment = `
+  <div class="one-comment">
+  <h6> ${currentComment.content} </h6>
+  </div>`
+  commentSection.insertAdjacentHTML('afterbegin', comment)
+  commentInput.value = '';
+  var ideaCardID = event.target.closest(".idea-card").dataset.id
+  for (var i = 0; i < list.length; i++){
+    if(list[i].id == ideaCardID){
+      list[i].comments.push(currentComment)
+      list[i].saveToStorage(list);
+    }
+  }
+  var button = event.target
+  button.disabled = true;
+}
+
+function printIdeaHTML(list, imgCardSrc){
```

Comment:
This was an awesome idea! Cleans up a lot of code when adding multiple variables.